ui/redux/actions/comments.js

```diff
@@ -96,20 +98,32 @@ export function doCommentList(
         return result;
       })
       .catch((error) => {
-        if (error.message === 'comments are disabled by the creator') {
-          dispatch({
-            type: ACTIONS.COMMENT_LIST_COMPLETED,
-            data: {
-              authorClaimId: authorChannelClaim ? authorChannelClaim.claim_id : undefined,
-              disabled: true,
-            },
-          });
-        } else {
-          devToast(dispatch, `doCommentList: ${error.message}`);
-          dispatch({
-            type: ACTIONS.COMMENT_LIST_FAILED,
-            data: error,
-          });
+        switch (error.message) {
+          case 'comments are disabled by the creator':
```

Comment:
where is this coming from? can/should it be a const?

Reply:
:+1: Yup, recently cleaned up to be constants in `master`.